config/Firebase/firebase.js

```diff
@@ -59,6 +59,19 @@ const Firebase = {
       .where('uid', '==', userId)
       .get()
   },
+  getUsers: (userId) => {
+    return firebase
+      .firestore()
+      .collection('users')
+      .get()
+  },
+  getUserData: (userId) => {
```

Comment:
This should probably be changed to getUser. You can see how Preston has changed his around on his branch